### ISSUE TITLE
Make library compatible with latest FEniCS version

### ISSUE THREAD

**Issue by finsberg** [finsberg/pulse#4]
`pulse` currently support FEniCS version 2017.x but it would be good if it also supported the latest version (which at the time writing is 2019.1.0).

**Comment by finsberg** [comment]
https://github.com/finsberg/pulse/tree/fenics2019

**Comment by finsberg** [comment]
Merged into master. All code is working in serial. A few things still don't work in parallel, but this can be a separate issue.
